www/configs/user-settings.json

```diff
@@ -73,7 +73,7 @@
     "other-options": {
       "title": "OTHER_OPTIONS",
       "type": "switch",
-      "settings": ["limit-change-log", "statistics"]
+      "settings": ["limit-change-log", "statistics", "intelligent-spacebar"]
```

Comment:
Small change, Instead of other options, can you please move this setting to App Settings.

Reply:
Sure